third_party/move/extensions/move-table-extension/src/lib.rs

```diff
@@ -685,13 +697,25 @@ fn get_table_handle(table: &StructRef) -> PartialVMResult<TableHandle> {
     Ok(TableHandle(handle))
 }
 
-fn serialize(layout: &MoveTypeLayout, val: &Value) -> PartialVMResult<Vec<u8>> {
-    val.simple_serialize(layout)
+fn serialize(
+    function_extension: &dyn FunctionValueExtension,
```

Comment:
nit: `function_value_extension` or short `fn_value_extension`.

Also, `FunctionValueExtension` can be short `FnValueExtension` but no strong opinions there

Reply:
Renamed to `function_value_extension` everywhere